File: output/llama3.1-8b/avatar/Python/Java/atcoder_ABC042_A.java
import java.util.Arrays;
import java.util.Scanner;

public class atcoder_ABC042_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String[] inputStr = scanner.nextLine().split(" ");
        int[] ABC = Arrays.stream(inputStr).mapToInt(Integer::parseInt).toArray();
        
        if (Arrays.stream(ABC).filter(i -> i == 5).count() == 2) {
            System.out.println((Arrays.stream(ABC).sum() == 17) ? "YES" : "NO");
        } else {
            System.out.println("NO");
        }
    }
}

File: output/llama3.1-8b/avatar/Java/Python/atcoder_ABC042_A.py
import sys
line = input()
list_ = line.split()
x, a, y = map(int, list_)
h = 'NO'
if x in [5, 7] or a in [5, 7] or y in [5, 7]:
    if x + y + a == 17:
        h = 'YES'
print(h)